package.json

```diff
@@ -5,34 +5,49 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "deploy:s3": "serverless client deploy --no-config-change --no-policy-change --no-cors-change",
+    "build:deploy": "npm run build && npm run deploy:s3",
```

Comment:
You probably forgot to remove this after adding "client:deploy" and "client:build:deploy"

Reply:
Fixed.